docs/howto/object-storage/s3/versioning.md

```diff
@@ -0,0 +1,74 @@
+# Object versioning
+
+> Object versioning requires that you configure your environment with
```

Comment:
@foadlind One more tiny nit: please put this reference at the top of the `expiry.md` file as well.

Reply:
Ack